enterprise/__init__.py

```diff
@@ -4,6 +4,6 @@
 
 from __future__ import absolute_import, unicode_literals
 
-__version__ = '0.1.2'
+__version__ = '0.1.3'
```

Comment:
MINOR updates are normally used for enhancements and fixes which do not break contracts (syntax) or alter existing behavior (semantics) in a backwards-incompatible way.  Breaking changes are represented by MAJOR version updates.